tests/generators/OperationGenerator.ts

```diff
@@ -84,12 +84,13 @@ export default class OperationGenerator {
    * Mainly used for testing.
    * @returns [publicKey, privateKey]
    */
-  public static async generateKeyPair (id: string): Promise<[PublicKeyModel, JwkEs256k]> {
+  public static async generateKeyPair (id: string, usage?: string[]): Promise<[PublicKeyModel, JwkEs256k]> {
     const [publicKey, privateKey] = await Jwk.generateEs256kKeyPair();
     const publicKeyModel = {
       id,
       type: 'Secp256k1VerificationKey2018',
-      jwk: publicKey
+      jwk: publicKey,
+      usage: usage || ['ops', 'general', 'auth']
```

Comment:
Likely that implementations that follow did core, and json-ld data model will not model things this way, but this is just one implementation.